torchvision/models/densenet.py

```diff
@@ -121,7 +121,7 @@ def __init__(
             )
             self.add_module('denselayer%d' % (i + 1), layer)
 
-    def forward(self, init_features: Tensor) -> Tensor:  # type: ignore[override]
+    def forward(self, init_features):
```

Comment:
Only the `type: ignore` comment has to go.

```suggestion
    def forward(self, init_features: Tensor) -> Tensor:
```

Reply:
Fixed!